### DISCUSSION TITLE
How do I use this?

### DISCUSSION THREAD

**Discussion by thepwrtank18** [Senth/minecraft-mod-manager#94] [Q&A] [answered]
I try using `pip install --user --upgrade minecraft-mod-manager`, and I do a command, and it says it can't find mmm.

**Comment by Senth** [comment, marked as answer]
If you're using Windows, you can take a look at [this issue](https://github.com/Senth/minecraft-mod-manager/issues/70) which describes how to do that 🙂 

If you're using Linux or OSX you have to add your user's bin dir to your PATH to .bashrc (or equivalent) 🙂 
```bash
export PATH="$PATH:$HOME/.local/bin"
```

**Reply by Senth** [reply]
In the future, there will be a wiki page detailing how to set it up correctly.
